tests.js

```diff
@@ -0,0 +1,88 @@
+require('required_env')([ 'AUTH_URL' ]);
+import Driver from './src';
+import test from 'blue-tape';
+import bandname from 'bandname';
+
+// Set to your auth api key
+const TEST_PASSWORD = 'hai';
```

Comment:
Ah, okay, that makes sense. In that case, though, we should take AUTH_PASSWORD (and in the future, AUTH_TOKEN) in through environment variables rather than hardcoding them.